composer.json

```diff
@@ -30,7 +30,8 @@
     "symfony/workflow": "^3.2|^4.0"
```

Comment:
We should allow symfony/workflow as of ^5.0 too

Reply:
I forgot to port this change from #16 :man_facepalming: